doc/nrf/app_dev/device_guides/nrf91/nrf91_cloud_connecting.rst

```diff
@@ -44,83 +44,90 @@ After logging in, you are taken to the dashboard view that displays your device
 
-     .. note::
+           .. figure:: images/nrfcloud_add_lte_device.png
+              :alt: nRF Cloud - Add LTE Device
 
-        The ownership code serves as a password and proves that you own the specific |DK|.
-        Therefore, do not share it with anyone.
+              nRF Cloud - Add LTE Device
 
-#. Click the :guilabel:`Add Device` button.
+           * **Device ID:** The device ID is composed of *nrf-* and the 15-digit :term:`International Mobile (Station) Equipment Identity (IMEI)` number that is printed on the label of your |DK|.
+             For example, *nrf-123456789012345*.
+             It is case sensitive, so make sure all the letters are lower-case.
+           * **PIN/HWID:** The ownership code is the PIN or the hardware ID of your device, and it is found on the label of your |DK|.
+             This is not the PIN code for your SIM card.
 
-   The **Do you need to activate an iBasis SIM?** pop-up opens.
+           If the label contains a PIN in addition to the IMEI number, enter this pin.
+           If it does not contain a PIN, enter the Hardware ID (HWID) HEX code, with or without colons.
+           For example, *AA:BB:CC:DD:EE:FF* or *AABBCCDDEEFF*.
 
-#. Click :guilabel:`Continue` and wait for the device to reconnect to nRF Cloud.
-   It is normal for the device to disconnect and reconnect multiple times during device provisioning.
+           .. note::
+              The ownership code serves as a password and proves that you own the specific |DK|.
+              Therefore, do not share it with anyone.
 
-   If you have not yet activated the iBasis SIM card that came with your |DK|, click :guilabel:`Activate iBasis SIM` instead, and follow the instructions.
+        #. Click the :guilabel:`Add Device` button.
 
-The |DK| is now added to your nRF Cloud account.
-This is indicated by the |led_publishing_data|, which shows that the device is publishing data.
+           The **Do you need to activate an iBasis SIM?** pop-up opens.
 
-On nRF Cloud, you can access the device by clicking :guilabel:`Devices` under :guilabel:`Device Management` in the navigation pane on the left.
+        #. Click :guilabel:`Continue` and wait for the device to reconnect to nRF Cloud.
+           It is normal for the device to disconnect and reconnect multiple times during device provisioning.
 
-.. nrf_cloud_add_device_end
+           If you have not yet activated the iBasis SIM card that came with your |DK|, click :guilabel:`Activate iBasis SIM` instead, and follow the instructions.
+
+           The |DK| is now added to your nRF Cloud account.
+           This is indicated by the |led_publishing_data|, which shows that the device is publishing data.
+
+           On nRF Cloud, you can access the device by clicking :guilabel:`Devices` under :guilabel:`Device Management` in the navigation pane on the left.
+
+        .. nrf_cloud_add_device_end
 
 .. _ug_nrf9160_gs_testing_gnss:
```

Comment:
The section below,

_Testing the GNSS functionality_ seems misplaced. The section is really on how to get connected to nrf cloud